### PR TITLE
StaticHomePage

### PR DESCRIPTION
**Pull request by calpuche**
Updated to have a Basic Home Page
Updated ReadMe

Preview for this branch: http://good-emporium-web-dev.s3-website.us-east-2.amazonaws.com/StaticHomePage/

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,5 +1,14 @@
 # web-ui
 
+#Dependencies
```
Comment (laconc) [inline README.md]: Can I assume that you're trying to create a header here? If so, Markdown requires a space after the pound sign. Additionally, I would suggest two pound signs as the title should be the biggest block on the page.

File: README.md [hunk 2]
```diff
@@ -1,5 +1,14 @@
...
+NPM- Installs with Node.js here: 
+https://nodejs.org/en/
```
Comment (laconc) [inline README.md]: We should be a bit opinionated here and recommend the latest v8 series, as it's the current Long-Term Support version, and it's the version that our CI/CD tool is using to build all the deployed code.

File: package.json [hunk 3]
```diff
@@ -38,6 +38,7 @@
   },
   "dependencies": {
     "react": "^16.4.1",
-    "react-dom": "^16.4.1"
+    "react-dom": "^16.4.1",
+    "react-scripts": "1.1.4"
```
Comment (laconc) [inline package.json]: Is `react-scripts` needed? Taking a look at it's [dependencies](https://github.com/firstlookmedia/react-scripts/blob/master/package.json), it is extremely heavy. Can we make do with pulling just the few packages from there that we'll be using? Taking a look at the current code, it looks like only the base React is used.

**Comment by justscribner** [comment]
I'm going to go ahead and merge so we have a starting point and we can do a dependency/module check when I get everything converted to components etc.

**Pull request merged**